### PR TITLE
Admission trial project for PR

### PR DESCRIPTION
**Pull request by SurafelGetachewT**
This was the admission trial project.
In this project:
1. we use HTML codes.
2. we didn't use any CSS.

### CODE REVIEW

**Review by Ginohmk — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, You are to make a review on this project, act as a reviewer and make a review, then  request for another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -78,8 +78,6 @@ <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
             </div>
       </form>  
     </div>
-
-
     <script  src="js/index.js"></script>
```
Comment (SurafelGetachewT) [inline index.html:81]: It is not recommended to add blank lines without a reason. Adding blank lines without a reason violates HTML best practices.

File: index.html [hunk 2]
```diff
@@ -23,8 +23,8 @@
     -->
     <div class="container">
       <header class="header">
-        <h1 id="title" class="text-center">freeCodeCamp Survey Form</h1>
-        <p id ="description">Thank you for taking the time to help us improve the platform</p>
+        <h1 id="title" class="text-center">FreeCodeCamp Survey Form</h1>
+        <p id ="description">Thank you for taking the time to help us improve the platform!</p>
```
Comment (SurafelGetachewT) [inline index.html:27]: I suggest you use a spaceless equal sign. An equal sign with a space makes your code inconsistent and violates HTML best practices.

**Review by SurafelGetachewT — COMMENTED** [review]
# Changes Required ♻️ 

You have done good so far!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966

# General Comments
 
1. You didn't use linters.
2. You didn't have CSS for your HTML.
3. Your repo is not documented professionally.

In addition, please check my comments below and make the required changes.

Submit for a re-review once you're done.

Enjoy coding

Thanks

**Comment by SurafelGetachewT** [comment]
Thanks a lot.

**Pull request merged**